extensions/jethub-burn.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Extension: jethub-burn
+# Automatically converts Armbian .img into burn image after build
+
+# Download and prepare tools
+function bootstrap_tools() {
+  local repo_url="https://github.com/jethome-iot/jethome-tools.git"
+  local ref="commit:87be932dceb6135c99dfc5a105a6345eff954f2c"
+  local name="jethub-burn"
+  local base="${SRC}/cache/sources/${name}"
+
+  display_alert "jethub-burn" "Fetching jethome-tools (${ref})..." "info"
+  fetch_from_repo "${repo_url}" "${name}" "${ref}"
+
+  local packer_path
+  packer_path="$(find "${base}" -maxdepth 12 -type f -path '*/tools/aml_image_v2_packer_new' -print -quit)"
```

Comment:
you cat set tool path to `${SRC}/cache/sources/jethome-tools/tools/aml_image_v2_packer_new`, so not use find